lib/traject/bibtex_config.rb

```diff
@@ -41,8 +41,8 @@
 to_fields %w(pub_year_isi pub_year_w_approx_isi), extract_bibtex(:year)
 to_field 'editor_ssim', extract_bibtex_field(:editor)
 to_field 'book_title_ssim', extract_bibtex_field(:booktitle)
-to_field 'pub_display', extract_bibtex_field(:journal)
-to_field 'pub_display', extract_bibtex_field(:publisher)
+to_field %w(pub_display pub_search), extract_bibtex_field(:journal)
+to_field %w(pub_display pub_search), extract_bibtex_field(:publisher)
```

Comment:
Assuming we leave the solr schema as it was, these could both just change to:

```suggestion
to_field 'pub_search', extract_bibtex_field(:journal)
to_field 'pub_search', extract_bibtex_field(:publisher)
```

Reply:
I think I understand your comments if we were to not make any changes to the schema at all, how might we streamline these values so everything gets added to pub_search (and then that gets copied to pub_display).  Maybe we could chat later.

Reply:
In the current production Solr schema `pub_search` is getting copied to `pub_display`. There's a similar pattern in place for several other `_search` fields that get copied to `_display` fields. I'm proposing to leave this behavior as it is in the Solr schema since it's a pattern. For bibtext items add the journal and publisher info to `pub_search` and let the Solr schema copyfield directive copy `pub_search` to the `copy_display` field. This way we don't change the existing pattern, don't have to change the schema, and don't have to change anything about the dor handling. Happy to huddle on this.